model_analyzer/record/record.py

```diff
@@ -136,3 +137,42 @@ def timestamp(self):
         """
 
         return self._timestamp
+
+    def __mul__(self, other):
+        """
+        Defines left multiplication for records
+        with floats or ints
+
+        Returns
+        -------
+        Record
+        """
+
+        if isinstance(other, int) or isinstance(other, float):
+            self._value = self._value * other
```

Comment:
Yes, I did this because I thought it would be hard to extract the underlying type, and justified it by saying that only temporaries (records that are not actual data) are being multiplied or divided anyway.